<!DOCTYPE html>
<!-- saved from url=(0037)https://www.addgene.org/crispr/guide/ -->
<html class=" js fontface generatedcontent inlinesvg mediaqueries display-table" lang="en"><!--<![endif]--><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        
        
        <meta http-equiv="X-UA-Compatible" content="IE=edge">

        
        

        <!-- before pagemeta block -->
        
        <!-- base pagemeta block -->
        
        
        
        
        <!-- after pagemeta block -->
        <!-- start Sekizai "meta" block -->
        
<link rel="canonical" href="https://www.addgene.org/crispr/guide/">


<meta name="msapplication-config" content="none">


<link rel="icon" type="image/png" href="https://www.addgene.org/headers/images/addgene-favicon.png">


<link rel="apple-touch-icon" href="https://www.addgene.org/headers/images/addgene-icon-square.png">


        <!-- end Sekizai "meta" block -->


        
        <!-- site-wide CSS, compressed -->
        <link rel="stylesheet" href="./Addgene_ CRISPR_Cas9 Guide_files/7705050bfa18.css" type="text/css">


        
        
        
        

        

        
        
        

        
        
        

        
        
        <!-- static CSS -->
        
        


        
        
        
        <!-- external CSS -->
        
        


        
        <!-- media.css -->
        
        

        


        <!-- base pagecss block -->
        
        
        
        










    



























        <!-- Sekizai "css" block -->
        
        
        
        <style>
            #ie7-register-link { display: none }
            html.lt-ie8 #register-link { display: none; }
            html.lt-ie8 #ie7-register-link { display: inline; }
        </style>
        

<style id="html-chunk-plugin-">
  .glyphicon-new-window {
    font-size: 10px;
}

/* Overwriting native bootstrap styling */

#spy-div {
    width: 82%
}

#scrollspy-div {
    width: 18%;
}

/* Styling and set up for the navigation panel */

body {
    data-spy: scroll;
    data-target: "#scrollspy-div";
    position: relative;
}

.affix-top {
    position:relative;
}

.affix {
  top: 40px;
  width: 150px;
}

@media (min-width: 1200px) {
  .affix {
    width: 150px;
  }         
}

.affix-bottom {
  width: 150px;
  position: absolute;
}

@media (min-width: 1200px) {
  .affix-bottom {
    width: 150px;
  }
}

#crispr-nav-panel {
    border: 1px solid #ddd; 
    border-radius: 4px; 
    padding: 5px 6px; 
    width: 150px;
    font-size: 12px;
}

#crispr-nav-panel > li > a {
    padding: 5px 5px;
}

/* Minor nav styling */
#crispr-nav-panel .nav a {
  font-weight: normal;
  font-size: .85em;
  padding: 5px 5px;
}

#crispr-nav-panel .nav .active a, #crispr-nav-panel .nav .active:hover a, #crispr-nav-panel .nav .active:focus a {
  font-weight: bold;
  padding-left: 10px;
  border-left: 1px dotted black;
}

.nav .active {
  font-weight: bold;
  background: #E2F1F6;
}

#crispr-nav-panel .nav {
  display: none;
}

#crispr-nav-panel .active .nav {
  display: block;
}

/* Styling of menu items */
#jumpto-title {
    font-weight: bolder;
    margin-bottom: 1em;
}

.menu-title {
    font-style: italic; 
    margin-bottom: 0.5em; 
    font-weight: bold;
}

/* Image styling */
#crispr-intro-img {
    margin: -90px -20px -90px -35px;
}

#knockout-img {
    margin: 5px -25px 5px 5px;
}

#nickase-img {
    margin: 5px -20px 5px 5px;
}

#hdr-img {
    margin: 5px -5px 5px 5px;
}

#ai-img {
    margin-top: 10px;
    margin-right: -25px;
    margin-left: 5px;
}

#screen-img {
    margin: -110px -100px -110px -120px;
}

#grna-design-img {
    margin-top: 5px;
    margin-right: -45px;
    margin-bottom: -150px;
}

.shortline {
    width: 70%;
    margin-top: 2em;
    border-top: dotted 1px;
}

.textspace {
    margin-top: 2.5em;
}

/* Table styling */

.tbl-col-20 {
    width: 20%;
}

.table-list li {
    margin-left: -20px;
    margin-bottom: 5px;
}

.table-list li:last-child {
    margin-bottom: -5px;
}

/* List styling */

#grna-design-list {
    font-weight: bold;
    list-style-type: upper-alpha;
}

.ol-unbolding {
    font-weight: normal;
}

.spacing-lists li {
    margin-bottom: 5px;
}

#edit-validate li {
    margin-top: 1.5em;
}

#pam-table tr td:first-child {
    font-style: italic;
}

</style>



        
        <!-- Sekizai "headjs" block start -->
        <iframe src="javascript:false" title="" style="display: none;" src="./Addgene_ CRISPR_Cas9 Guide_files/saved_resource.html"></iframe><script src="./Addgene_ CRISPR_Cas9 Guide_files/analytics.js.download"></script><script type="text/javascript" src="./Addgene_ CRISPR_Cas9 Guide_files/cb6e9eee10ad.js.download"></script>

        <!-- Sekizai "headjs" block end -->

		<title>Addgene: CRISPR/Cas9 Guide</title>
        <meta name="description" content="Addgene&#39;s guide to using CRISPR plasmids in your lab, from knockouts to pooled library screening. ">
        <meta name="keywords" content="">
    </head>
    <body class="">
        
        
        
        
        
        


        
        
        
        
        

        

        <header>
<div id="addgene-header">
  <div class="container">
    <div id="top-row">

      
      <div class="logo">
        <a href="https://www.addgene.org/">
          <img src="./Addgene_ CRISPR_Cas9 Guide_files/addgene-logo.png" alt="Addgene">
        </a>
      </div>

      
      
      <div class="login-new-user text-center">
        <a href="https://www.addgene.org/users/login/?next=/crispr/guide/">Login</a>
        |
        <!-- IE 7 can't handle the user-reg javascript, and we don't have -->
        <!-- time to figure out a workaround just for it. -->
        
        
        <a id="ie7-register-link" href="https://www.addgene.org/browser/">Create Account</a>
        
        <a id="register-link" href="https://www.addgene.org/users/register">Create Account</a>
      </div>
      

      
      <div class="search">
        <form id="search-form" method="GET" action="https://www.addgene.org/search/advanced/">
          <div class="input-container">
            <input id="search_q_id" type="text" name="q" value="" placeholder="Search for plasmids">
          </div>
          
          <button type="submit" id="search_submit_id">
            <span class="glyphicon glyphicon-search"></span>
          </button>
        </form>
      </div> 
    </div> 
  </div> 

  
  <nav id="main-nav">
    <div class="container">
        <ul class="nav nav-pills nav-justified">
          
          <li class="dropdown">
            
            <a href="https://www.addgene.org/crispr/guide/#" class="dropdown-toggle" data-toggle="dropdown">
              Find Plasmids <b class="caret"></b>
            </a>
            <ul class="dropdown-menu">
              <li>
                <a href="https://www.addgene.org/search/advanced/">Search</a>
              </li>
              
              <li><a href="https://www.addgene.org/browse/">Browse</a></li>
              <li>
                
                <a href="https://www.addgene.org/viral-service/">
                  Viral Service
                </a>
              </li>
              <li>
                
                <a href="https://www.addgene.org/special-collections/">
                  Special Collections
                </a>
              </li>
              <li>
                
                <a href="https://www.addgene.org/popular-plasmids/">
                  Popular Plasmids
                </a>
              </li>
              <li><a href="https://www.addgene.org/alerts/">Alerts</a></li>
            </ul>
          </li>

          
          <li class="dropdown">
            
            
            <a href="https://www.addgene.org/crispr/guide/#" class="dropdown-toggle" data-toggle="dropdown" id="deposit-dropdown">
              Deposit Plasmids <b class="caret"></b>
            </a>
            <ul class="dropdown-menu">
              
              
              <li><a href="https://www.addgene.org/deposit/" id="page-menu-deposit">Why Deposit</a></li>
              <li>
                
                <a href="https://www.addgene.org/depositing/start-deposit/" id="page-deposit-menu-add-article">Start Deposit</a>
              </li>
              <li>
                <a href="https://www.addgene.org/plasmids/plasmids/">My Plasmids</a>
              </li>
              <li>
                
                <a href="https://www.addgene.org/plasmids/articles/" id="page-deposit-menu-my-articles">My Articles</a>
              </li>
              <li>
                
                <a href="https://www.addgene.org/techtransfer/tto/#deposit-process">
                  Deposit Agreements
                </a>
              </li>
              <li>
                <a href="https://www.addgene.org/users/rewards/">
                  Rewards Points
                </a>
              </li>
            </ul>
          </li>

          
          <li class="dropdown">
            
            <a href="https://www.addgene.org/crispr/guide/#" class="dropdown-toggle" data-toggle="dropdown">
              How to Order <b class="caret"></b>
            </a>
            <ul class="dropdown-menu">
              
              <li><a href="https://www.addgene.org/ordering/">Ordering Instructions</a></li>
              
              <li><a href="https://www.addgene.org/ordering/purchase-orders/">Purchase Orders</a></li>
              
              <li>
                <a href="https://www.addgene.org/ordering/#pricing">Pricing and Payment</a>
              </li>
              
              <li><a href="https://www.addgene.org/ordering/permits/">Import Permits</a></li>
              
              <li><a href="https://www.addgene.org/ordering/#ship">Shipping</a></li>
              
              <li><a href="https://www.addgene.org/ordering/international/">International Customers</a></li>
              
              <li><a href="https://www.addgene.org/industry/">Industry Customers</a></li>
              
              <li><a href="https://www.addgene.org/techtransfer/#request">Material Transfer Agreements</a></li>
            </ul>
          </li>

          
          <li class="dropdown">
            
            <a href="https://www.addgene.org/crispr/guide/#" class="dropdown-toggle" data-toggle="dropdown">
              Plasmid Reference <b class="caret"></b>
            </a>
            <ul class="dropdown-menu">
              
              <li><a href="https://www.addgene.org/plasmid-reference/">Overview</a></li>
              
              <li><a href="https://www.addgene.org/recipient-instructions/">Recipient Instructions</a></li>
              <li><a href="https://www.addgene.org/analyze-sequence/">Analyze Sequence</a></li>
              <li><a href="https://www.addgene.org/vector-database/">Vector Database</a></li>
              
              <li><a href="https://www.addgene.org/protocols/">Protocols</a></li>
              
              <li><a href="https://www.addgene.org/mol-bio-reference/">Molecular Biology Reference</a></li>
            </ul>
          </li>

          
          <li class="dropdown">
            
            <a href="https://www.addgene.org/crispr/guide/#" class="dropdown-toggle" data-toggle="dropdown">
              About Addgene <b class="caret"></b>
            </a>
            <ul class="dropdown-menu">
              
              <li><a href="https://www.addgene.org/mission/">Mission</a></li>
              
              <li><a href="https://www.addgene.org/contact/">Contact</a></li>
              
              <li><a href="https://www.addgene.org/management/">Leadership</a></li>
              
              <li><a href="https://www.addgene.org/careers/">Careers</a></li>
              
              <li><a href="https://www.addgene.org/testimonials/">Testimonials</a></li>
              
              <li><a href="https://www.addgene.org/techtransfer/">Technology Transfer</a></li>
              
              <li><a href="https://www.addgene.org/sponsors/">Sponsors</a></li>
              
              <li><a href="https://www.addgene.org/links/">Links</a></li>
              
              <li><a href="https://www.addgene.org/faq/">FAQ</a></li>
              
              <li><a href="https://www.addgene.org/news/">News</a></li>
              <li><a href="http://blog.addgene.org/" target="_blank" rel="noopener noreferrer">Blog</a></li>
            </ul>
          </li>
        </ul>
    </div>
  </nav>
</div> 






<div id="messages" class="container">

  

  






  
      <div id="addgene-messages-4" class="alert alert-info alert-dismissable">
        <button id="addgene-messages-close-4" onclick="jQuery.cookie(&#39;addgene-messages-4&#39;, &#39;4&#39;, { path: &#39;/&#39;, expires: 365 })" type="button" class="close" data-dismiss="alert" aria-hidden="true">Close
        </button>
          
            This website uses cookies to ensure you get the best experience. By continuing the use this site, you agree to the use of cookies.
          
      </div>
  





  





















<div id="message-browser-wontwork" class="alert alert-danger">
    


    <!-- This is the message presented to users of browsers missing feature we really require (like Javascript). -->

<p>
  Please note: Your browser does not support the features used on Addgene's website. You may not be able to create an account or request plasmids through this website until you upgrade your browser.
  <a href="https://www.addgene.org/browser/">Learn more</a>
</p>


</div>


<div id="message-browser-old" class="alert alert-warning">
    


    <!-- This message appears on browser we know have some issues with how they display our site, but shouldn't prevent it's use. -->

<p>
  Please note: Your browser does not fully support some of the features used on Addgene's website. If you run into any problems registering, depositing, or ordering please contact us at help@addgene.org.
  <a href="https://www.addgene.org/browser/">Learn more</a>
</p>


</div>


  






  
</div>

</header>

        <div id="page-body">

        
        
        
        
        
        
        
        
        
        
        
        
        
        
        <main>
            
            <div class="container">
                






<!--
Styling in PageCSS for sticky nav bar, image spacing, list items
Also, javascript for sticky nav bar
NW Oct 2015
--><!-- This file was automatically modified on 2017-01-03 -->
 
  <div class="breadcrumbs row">
   <div class="col-xs-12">
    <ul>
     <li><a href="https://www.addgene.org/genome-engineering/">Genome Engineering</a></li>
     <li><a href="https://www.addgene.org/crispr/">CRISPR</a></li>
     <li>CRISPR Guide</li>
    </ul>
   </div>
  </div>
  <div class="row">
   <div class="col-xs-8">
    <h1><span>
      <img class="filer_image " alt="CRISPR header icon" src="./Addgene_ CRISPR_Cas9 Guide_files/crispr_header_icon.gif__68x50_q85_subsampling-2_upscale.png" height="50">

     </span>
     CRISPR/Cas9 Guide</h1>
   </div>
   <div class="col-xs-4">
    <div class="pull-right" id="CRISPR-eBook-CTA" style="padding-top: 10px;">
     <a href="http://info.addgene.org/download-addgenes-ebook-crispr-101-1st-edition?__hssc=246494683.32.1454438085534&amp;__hstc=246494683.25061f05a3c348a1c72b4bc904efdfc9.1433249788495.1454434944296.1454438085534.476&amp;__hsfp=&amp;hsCtaTracking=def26d9c-3f9c-4d7b-b065-3e0d0e24669b%7Ceb8f44bc-44a1-4a74-b795-fc62e4bd999b/">
      <img class="filer_image right" alt="CRISPR 101 eBook CTA" src="./Addgene_ CRISPR_Cas9 Guide_files/crispr-ebook-cta.png__225x57_q85_subsampling-2_upscale.png" width="225">

     </a>
    </div>
   </div>
  </div>
  <hr>
  <!-- Nav sidebar -->
  <div class="col-xs-3 pull-right" id="scrollspy-div">
   <!-- major nav -->
   <ul class="nav hidden-xs hidden-sm affix" data-spy="affix" id="crispr-nav-panel">
    <li id="jumpto-title">
     Jump To:
    </li>
    <li class="menu-title">
     CRISPR Guide
    </li>
    <li class=""><a href="https://www.addgene.org/crispr/guide/#overview">Overview</a></li>
    <li class=""><a href="https://www.addgene.org/crispr/guide/#ko-generation">Knock-out Generation</a></li>
    <li class="active"><a href="https://www.addgene.org/crispr/guide/#nickase">Nickase</a></li>
    <li><a href="https://www.addgene.org/crispr/guide/#hdr">HDR</a></li>
    <li><a href="https://www.addgene.org/crispr/guide/#activation-repress">Activation or Repression</a></li>
    <li><a href="https://www.addgene.org/crispr/guide/#multiplex">Multiplex Genome Engineering</a></li>
    <li><a href="https://www.addgene.org/crispr/guide/#genome-screen">Genome-wide Screening</a><ul class="nav"><!-- minor nav --><li><a href="https://www.addgene.org/crispr/guide/#what-are-pooled-libraries">What are pooled libraries?</a></li>
      <li><a href="https://www.addgene.org/crispr/guide/#library-use">How do you use a library?</a></li>
      <li><a href="https://www.addgene.org/crispr/guide/#screen-results">Screening results</a></li>
     </ul>
     <!-- End minor nav -->
    </li>
    <li><a href="https://www.addgene.org/crispr/guide/#additional-cas9-use">Additional Cas9 Uses</a><ul class="nav"><!-- minor nav --><li><a href="https://www.addgene.org/crispr/guide/#fluoro-tag">Fluorophore Tags</a></li>
      <li><a href="https://www.addgene.org/crispr/guide/#genomic-purify">Purify Genomic Regions</a></li>
     </ul>
     <!-- End minor nav -->
    </li>
    <li><a href="https://www.addgene.org/crispr/guide/#alt-cas9">Cas9 Alternatives</a></li>
    <hr>
    <li class="menu-title">
     Plan Your Experiment
    </li>
    <!-- Major Nav -->
    <li><a href="https://www.addgene.org/crispr/guide/#plan-experiment">Overview</a></li>
    <li><a href="https://www.addgene.org/crispr/guide/#select-goal">Select Genetic Manipulation</a></li>
    <li><a href="https://www.addgene.org/crispr/guide/#select-model">Select Expression System</a></li>
    <li><a href="https://www.addgene.org/crispr/guide/#design-grna">Design gRNA</a><ul class="nav"><!-- Minor nav --><li><a href="https://www.addgene.org/crispr/guide/#know-cells">Know your Cell Line</a></li>
      <li><a href="https://www.addgene.org/crispr/guide/#select-gene">Select Gene</a></li>
      <li><a href="https://www.addgene.org/crispr/guide/#select-grna">Select gRNA</a></li>
      <li><a href="https://www.addgene.org/crispr/guide/#clone-grna">Clone gRNA</a></li>
      <li><a href="https://www.addgene.org/crispr/guide/#reagent-delivery">Delivery of gRNA &amp; Cas9</a></li>
      <li><a href="https://www.addgene.org/crispr/guide/#validate-results">Validate Results</a></li>
     </ul>
     <!-- end minor nav -->
    </li>
    <hr>
    <li class="menu-title">
     Resources
    </li>
    <li><a href="https://www.addgene.org/crispr/guide/#web-references">Web References</a></li>
    <li><a href="https://www.addgene.org/crispr/guide/#pam-table">PAM Sequences</a></li>
    <li><a href="https://www.addgene.org/crispr/guide/#glossary">Glossary</a></li>
    <li><a href="https://www.addgene.org/crispr/guide/#publications">Publications</a></li>
   </ul>
   <!-- End major nav -->
  </div>
  <div class="col-xs-9" id="spy-div">
   <section id="overview">
    <h2>Overview of CRISPR/Cas9</h2>
    <p>
     The <u>C</u> lustered <u>R</u> egularly <u>I</u> nterspaced <u>S</u> hort <u>P</u> alindromic <u>R</u> epeats (CRISPR) Type II system is a bacterial immune system that has been modified for genome engineering (see
     <a href="https://www.addgene.org/crispr/reference/history/">
      CRISPR history
     </a>
     ). Prior to CRISPR/Cas9,
     <a href="https://www.addgene.org/genome-engineering/">
      genome engineering approaches
     </a>
     , like zinc finger nucleases (ZFNs) or transcription-activator-like effector nucleases (TALENs), relied upon the use of customizable DNA-binding protein nucleases that required scientists to design and generate a new nuclease-pair for every genomic target. Largely due to its simplicity and adaptability, CRISPR has rapidly become one of the most popular approaches for genome engineering.
    </p>
    <div>
     <div class="pull-right" id="crispr-intro-img">
      <img class="filer_image " alt="CRISPR image of different functions" src="./Addgene_ CRISPR_Cas9 Guide_files/crispr-intro.png__420x453_q85_subsampling-2_upscale.png" width="420">

     </div>
     <p>
      CRISPR consists of two components: a “guide” RNA (gRNA) and a non-specific CRISPR-associated endonuclease (Cas9). The gRNA is a short synthetic RNA composed of a “scaffold” sequence necessary for Cas9-binding and a user-defined ∼20 nucleotide “spacer” or “targeting” sequence which defines the genomic target to be modified. Thus, one can change the genomic target of Cas9 by simply changing the targeting sequence present in the gRNA. CRISPR was originally employed to “knock-out” target genes in various cell types and organisms, but modifications to the Cas9 enzyme have extended the application of CRISPR to selectively activate or repress target genes, purify specific regions of DNA, and even image DNA in live cells using fluorescence microscopy. Furthermore, the ease of generating gRNAs makes CRISPR one of the most scalable genome editing technologies and has been recently utilized for genome-wide screens.
     </p>
     <p>
      This guide will provide a basic understanding of CRISPR/Cas9 biology, introduce the various applications of CRISPR, and help you get started using CRISPR/Cas9 in your own research.
     </p>
    </div>
   </section>
   <section id="ko-generation">
    <h2>Generating a Knock-out Using CRISPR/Cas9</h2>
    <div>
     <div class="pull-right" id="knockout-img">
      <img class="filer_image " alt="CRISPR Knockout Diagram" src="./Addgene_ CRISPR_Cas9 Guide_files/knockout-generation.png__332x480_q85_subsampling-2_upscale.png" height="480">

     </div>
     <p>
      CRISPR/Cas9 can be used to generate knock-out cells or animals by co-expressing a gRNA specific to the gene to be targeted and the endonuclease Cas9. The genomic target can be any ∼20 nucleotide DNA sequence, provided it meets two conditions:
     </p>
     <ol>
      <li>The sequence is unique compared to the rest of the genome.</li>
      <li>The target is present immediately upstream of a <u>P</u> rotospacer <u>A</u> djacent <u>M</u> otif (PAM).</li>
     </ol>
     <p>
      The PAM sequence is absolutely necessary for target binding and the exact sequence is dependent upon the species of Cas9 (5′ NGG 3′ for <em>Streptococcus pyogenes</em> Cas9). We will focus on Cas9 from <em>S. pyogenes</em> as it is currently the most widely used in genome engineering (
      <a href="https://www.addgene.org/crispr/guide/#pam-table">
       see additional species of Cas9 and corresponding PAM sequences here
      </a>
      ). Once expressed, the Cas9 protein and the gRNA form a riboprotein complex through interactions between the gRNA “scaffold” domain and surface-exposed positively-charged grooves on Cas9. Cas9 undergoes a conformational change upon gRNA binding that shifts the molecule from an inactive, non-DNA binding conformation, into an active DNA-binding conformation. Importantly, the  “spacer” sequence of the gRNA remains free to interact with target DNA. The Cas9-gRNA complex will bind any genomic sequence with a PAM, but the extent to which the gRNA spacer matches the target DNA determines whether Cas9 will cut. Once the Cas9-gRNA complex binds a putative DNA target, a “seed” sequence at the 3′ end of the gRNA targeting sequence begins to anneal to the target DNA. If the seed and target DNA sequences match, the gRNA will continue to anneal to the target DNA in a 3′ to 5′ direction.
     </p>
     <p>
      Cas9 will only cleave the target if sufficient homology exists between the gRNA spacer and target sequences. The “zipper-like” annealing mechanics may explain why mismatches between the target sequence in the 3′ seed sequence completely abolish target cleavage, whereas mismatches toward the 5′ end are permissive for target cleavage. The Cas9 nuclease has two functional endonuclease domains: RuvC and HNH. Cas9 undergoes a second conformational change upon target binding that positions the nuclease domains to cleave opposite strands of the target DNA. The end result of Cas9-mediated DNA cleavage is a double strand break (DSB) within the target DNA (∼3-4 nucleotides upstream of the PAM sequence).
     </p>
    </div>
    <p>
     The resulting DSB is then repaired by one of two general repair pathways:
    </p>
    <ol>
     <li>The efficient but error-prone Non-Homologous End Joining (NHEJ) pathway</li>
     <li>The less efficient but high-fidelity Homology Directed Repair (HDR) pathway</li>
    </ol>
    <p>
     The NHEJ repair pathway is the most active repair mechanism, capable of rapidly repairing DSBs, but frequently results in small nucleotide insertions or deletions (InDels) at the DSB site. The randomness of NHEJ-mediated DSB repair has important practical implications, because a population of cells expressing Cas9 and a gRNA will result in a diverse array of mutations (
     <a href="https://www.addgene.org/crispr/guide/#plan-experiment">
      for more information, jump to Plan Your Experiment
     </a>
     ). In most cases, NHEJ gives rise to small InDels in the target DNA which result in in-frame amino acid deletions, insertions, or frameshift mutations leading to premature stop codons within the open reading frame (ORF) of the targeted gene. Ideally, the end result is a loss-of-function mutation within the targeted gene;  however, the “strength” of the knock-out phenotype for a given mutant cell is ultimately determined by the amount of residual gene function.
    </p>
    <p> <em>Browse Plasmids:</em> <a href="https://www.addgene.org/crispr/cut/">
      Double Strand Break (Cut)
     </a>
    </p>
   </section>
   <section id="nickase">
    <h2>Enhancing Specificity with Cas9 Nickase</h2>
    <div>
     <div class="pull-right" id="nickase-img">
      <img class="filer_image " alt="Diagram of CRISPR knockout via nickase" src="./Addgene_ CRISPR_Cas9 Guide_files/knockout-nickase.png__353x470_q85_subsampling-2_upscale.png" height="470">

     </div>
     <p>
      CRISPR/Cas9 is highly specific when gRNAs are designed correctly, but specificity is still a major concern, particularly as CRISPR is being developed for clinical use. The specificity of the CRISPR system is determined in large part by how specific the gRNA targeting sequence is for the genomic target compared to the rest of the genome. Ideally, a gRNA targeting sequence will have perfect homology to the target DNA with no homology elsewhere in the genome. Realistically, a given gRNA targeting sequence will have additional sites throughout the genome where partial homology exists. These sites are called “off-targets” and need to be considered when designing a gRNA for your experiment (
      <a href="https://www.addgene.org/crispr/guide/#design-grna">
       more information on gRNA design can be found in the below Plan Your Experiment section
      </a>
      ).
     </p>
     <p>
      In addition to optimizing gRNA design, specificity of the CRISPR system can also be increased through modifications to Cas9 itself. As discussed previously, Cas9 generates double strand breaks (DSBs) through the combined activity of two nuclease domains, RuvC and HNH. The exact amino acid residues within each nuclease domain that are critical for endonuclease activity are known (D10A for HNH and H840A for RuvC in <em>S. pyogenes</em> Cas9) and modified versions of the Cas9 enzyme containing only one active catalytic domain (called “Cas9 nickase”) have been generated. Cas9 nickases still bind DNA based on gRNA specificity, but nickases are only capable of cutting one of the DNA strands, resulting in a “nick”, or single strand break, instead of a DSB. DNA nicks are rapidly repaired by HDR (homology directed repair) using the intact complementary DNA strand as the template (
      <a href="https://www.addgene.org/crispr/guide/#hdr">
       jump to our HDR section for more details
      </a>
      ). Thus, two nickases targeting opposite strands are required to generate a DSB within the target DNA (often referred to as a “double nick” or “dual nickase” CRISPR system). This requirement dramatically increases target specificity, since it is unlikely that two off-target nicks will be generated within close enough proximity to cause a DSB. Therefore, if specificity and reduced off-target effects are crucial, consider using the dual nickase approach to create a double nick-induced DSB. The nickase system can also be combined with HDR-mediated gene editing for highly specific gene edits.
     </p>
    </div>
    <p> <em>Browse Plasmids:</em> <a href="https://www.addgene.org/crispr/nick/">
      Single Strand Break (Nick)
     </a>
    </p>
   </section>
   <section id="hdr">
    <h2>Making Precise Modifications Using Homology Directed Repair (HDR)</h2>
    <p>
     While NHEJ-mediated DSB repair is imperfect and often results in disruption of the open reading frame of the gene, Homology Directed Repair (HDR) can be used to generate specific nucleotide changes (also known as gene “edits”) ranging from a single nucleotide change to large insertions (e.g. addition of a fluorophore or tag).
    </p>
    <div>
     <div class="pull-right" id="hdr-img">
      <img class="filer_image " alt="CRISPR diagram of homology direct repair or HDR" src="./Addgene_ CRISPR_Cas9 Guide_files/gene_editing-hdr2.png__367x480_q85_subsampling-2_upscale.png" height="480">

     </div>
     <p>
      In order to utilize HDR for gene editing, a DNA “repair template” containing the desired sequence must be delivered into the cell type of interest with the gRNA(s) and Cas9 or Cas9 nickase. The repair template must contain the desired edit as well as additional homologous sequence immediately upstream and downstream of the target (termed left &amp; right homology arms). The length and binding position of each homology arm is dependent on the size of the change being introduced. The repair template can be a single stranded oligonucleotide, double-stranded oligonucleotide, or double-stranded DNA plasmid depending on the specific application. It is worth noting that the repair template must lack the PAM sequence that is present in the genomic DNA, otherwise the repair template becomes a suitable target for Cas9 cleavage. For example, the PAM could be mutated such that it is no longer present, but the coding region of the gene is not affected (i.e. a silent mutation).
     </p>
    </div>
    <p>
     The efficiency of HDR is generally low (&lt;10% of modified alleles) even in cells that express Cas9, gRNA and an exogenous repair template.  For this reason, many laboratories are attempting to artificially enhance HDR by synchronizing the cells within the cell cycle stage when HDR is most active, or by chemically or genetically inhibiting genes involved in NHEJ. The low efficiency of HDR has several important practical implications. First, since the efficiency of Cas9 cleavage is relatively high and the efficiency of HDR is relatively low, a portion of the Cas9-induced DSBs will be repaired via NHEJ. In other words, the resulting population of cells will contain some combination of wild-type alleles, NHEJ-repaired alleles, and/or the desired HDR-edited allele. Therefore, it is important to confirm the presence of the desired edit experimentally, and if necessary, isolate clones containing the desired edit (
     <a href="https://www.addgene.org/crispr/guide/#validate-results">
      see our validation section in Plan Your Experiment
     </a>
     ).
    </p>
   </section>
   <section id="activation-repress">
    <h2>Activation or Repression of Target Genes Using CRISPR/Cas9</h2>
    <div>
     <div class="pull-right" id="ai-img">
      <img class="filer_image " alt="Diagram of CRISPR activation or repression" src="./Addgene_ CRISPR_Cas9 Guide_files/crisprai-2.png__280x450_q85_subsampling-2_upscale.png" width="280" height="450">

     </div>
     <p>
      The CRISPR/Cas system is a remarkably flexible tool for genome manipulation. A unique feature of Cas9 is its ability to bind target DNA independently of its ability to cleave target DNA. Specifically, both RuvC- and HNH- nuclease domains can be rendered inactive by point mutations (D10A and H840A in SpCas9), resulting in a nuclease dead Cas9 (dCas9) molecule that cannot cleave target DNA. The dCas9 molecule retains the ability to bind to target DNA based on the gRNA targeting sequence. The first experiments using dCas9 in bacteria demonstrated that targeting dCas9 to transcriptional start sites was sufficient to “repress” or “knock-down” transcription by blocking transcription initiation. Furthermore, dCas9 can be tagged with transcriptional repressors or activators, and targeting these dCas9 fusion proteins to the promoter region results in robust transcription repression or activation of downstream target genes. The simplest dCas9-based activators and repressors consist of dCas9 fused directly to a single transcriptional activator, <b>A</b> (e.g. VP64) or transcriptional repressors, <b>R</b> (e.g. KRAB; see <b>A</b> in Figure to the right). Additionally, more elaborate activation strategies have been developed which result in greater activation of target genes in mammalian cells. These include: co-expression of epitope-tagged dCas9 and antibody-activator effector proteins (e.g. SunTag system, <b>B</b> ), dCas9 fused to several different activation domains in series (e.g. dCas9-VPR, <b>C</b> ) or co-expression of dCas9-VP64 with a “modified scaffold” gRNA and additional RNA-binding “helper activators” (e.g. SAM activators, <b>D</b> ). Importantly, unlike the genome modifications induced by Cas9 or Cas9 nickase, dCas9-mediated gene activation or repression is reversible, since it does not permanently modify the genomic DNA.
     </p>
    </div>
    <p> <em>Browse Plasmids:</em> <a href="https://www.addgene.org/crispr/activate/">
      Activate
     </a>
     ,
     <a href="https://www.addgene.org/crispr/interfere/">
      Repress/Interfere
     </a>
    </p>
   </section>
   <section id="multiplex">
    <h2>Multiplex Genome Engineering with CRISPR/Cas9</h2>
    <p>
     Expressing several gRNAs off of the same plasmid ensures that every cell that takes up a plasmid expresses all of the desired gRNAs and increases the likelihood that all desired genomic edits will be carried out by Cas9. Such “multiplex” CRISPR applications include:
    </p>
    <ul>
     <li>The use of dual nickases to generate a knock-out or edit a gene</li>
     <li>Using Cas9 to generate large genomic deletions</li>
     <li>Modifying multiple different genes at once</li>
    </ul>
    <p>
     Current multiplex CRISPR systems enable researchers to target anywhere from 2 to 7 genetic loci by cloning multiple gRNAs into a single plasmid. These multiplex gRNA vectors can conceivably be combined with any of the aforementioned Cas9-derivatives to not only knock-out target genes, but activate or repress target genes as well.
    </p>
    <p> <em>Browse Plasmids:</em> <a href="https://www.addgene.org/crispr/empty-grna-vectors/">
      Multiplex gRNA vectors
     </a>
    </p>
   </section>
   <section id="genome-screen">
    <h2>Genome-wide Screens Using CRISPR/Cas9</h2>
    <p>
     The ability to semi-automatically design and synthesize gRNAs to mutate, activate, or repress almost any genomic locus makes the CRISPR/Cas9 the ideal genome engineering system for large-scale forward genetic screening. Forward genetic screens are particularly useful for studying diseases or phenotypes for which the underlying genetic cause is not known. In general, the goal of a genetic screen is to generate a large population of cells with mutations in a wide variety of genes and use these mutant cells to identify the genetic perturbations that result in a desired phenotype. Before CRISPR/Cas9, genetic screens relied heavily on shRNA-based screens, which are prone to off-target effects and may result in false negatives due to incomplete knock-down of target genes. The CRISPR system, in contrast, is capable of making highly specific, permanent genetic modifications in target genes. The CRISPR system has already been used to screen for novel genes that regulate known phenotypes, including resistance to chemotherapy drugs, resistance to toxins, cell viability, and tumor metastasis. Currently, the most popular method for conducting genome-wide screens using CRISPR/Cas9 involves the use of “pooled” lentiviral CRISPR libraries.
    </p>
    <h3 id="what-are-pooled-libraries">
     What are pooled lentiviral CRISPR libraries?</h3>
    <p>
     Pooled lentiviral CRISPR libraries (heretofore referred to as CRISPR libraries) are a heterogenous population of lentiviral transfer vectors, each containing  an individual gRNA targeting a single gene in a given genome.
    </p>
    <div>
     <div class="pull-right" id="screen-img">
      <img class="filer_image " alt="Screen-flowchart.png" src="./Addgene_ CRISPR_Cas9 Guide_files/screen-flowchart.png__550x594_q85_subsampling-2_upscale.png" width="550">

     </div>
     <p>
      Guide RNAs are designed <em>in silico</em> and synthesized ( <b>see A</b> in figure to the right), then cloned in a pooled format into lentiviral transfer vectors <b>B</b> . CRISPR libraries have been designed for most of the common CRISPR applications including genetic knock-out and activation or repression for both human and mouse genes. Although each library is different, there are several features that are common across most CRISPR libraries. Each library typically contains ∼3-6 gRNAs per gene to ensure modification of every target gene. Libraries can target anywhere from a single class of genes up to every gene in the genome. Thus, CRISPR libraries contain thousands of unique gRNAs targeting a wide variety of genes. Guide RNA design for CRISPR libraries follows the same general principles as designing a gRNA for a specific target. Target sequences must be unique compared to the rest of the genome and be located just upstream of a PAM sequence. Obviously, the exact region of the gene to be targeted may vary depending on the specific application (5′ constitutively expressed exons for knock-out libraries, or the promoter region for activation and repression libraries). For some libraries, Cas9 (or Cas9 derivative) is included on the gRNA-containing plasmid; for others, they must be delivered to the cells separately.
     </p>
    </div>
    <h3 id="library-use">
     How does one use a CRISPR library?</h3>
    <p>
     All of the CRISPR libraries available through Addgene follow the same general experimental protocol. In most cases, the CRISPR library will be shipped at a concentration that is too low to be used in experiments. Thus, the first step in using your library is to “amplify” the library ( <b>C</b> in above figure) such that the total amount of DNA is increased but the “representation” (i.e. the relative percentage of each gRNA with the library) is maintained. Once the library has been amplified and the representation checked using next-generation sequencing (NGS), the next step is to generate lentivirus containing the entire CRISPR library <b>D</b> . Mutant cells are then generated by transducing Cas9-expressing cells (or wild-type cells for libraries containing Cas9 and the gRNA) with the lentiviral library <b>E</b> . In screens where you are measuring the loss of gRNAs from a final population (i.e. negative selection survival screens) you need to use NGS to identify the gRNAs present in the initial mutant cell population prior to selection. Alternatively, for positive screens such as drug-screens, you can treat your mutant cells with drug, or control and directly compare the gRNA distribution at the end of the screen <b>F</b> . It is important to remember that analysis of relevant genes (“hits”) at the end of your screen requires the use of NGS.
    </p>
    <h3 id="screen-results">
     What can screens tell you?</h3>
    <p>
     As noted at the beginning of this section, forward genetic screens are most useful for situations in which the physiology or cell biology behind a particular phenotype or disease is well understood, but the underlying genetic causes are unknown. Therefore, genome-wide screens using CRISPR libraries are a great way to gather unbiased information regarding which genes, if any, play a causal role in a given phenotype. With any experiment, it is important to be sure that the hits you identify are actually important for your phenotype. This is typically carried out by testing the gRNAs identified in your screen individually to ensure that the genetic modification reproduces the phenotype you screened for in the first place.
    </p>
    <p>
     Find more information on what you should consider before starting your pooled library experiment in
     <a href="https://www.ncbi.nlm.nih.gov/pubmed/27366891" rel="noopener noreferrer" target="_blank">
      <span class="glyphicon glyphicon-new-window">
      </span>
      “Practical Considerations for Using Pooled Lentiviral CRISPR Libraries”
     </a>
     (McDade et al., 2016).
    </p>
    <p> <em>Browse Libraries:</em> <a href="https://www.addgene.org/crispr/libraries/">
      CRISPR Pooled Libraries
     </a>
    </p>
   </section>
   <section id="additional-cas9-use">
    <h2>Additional Uses of Cas9</h2>
    <h3 id="fluoro-tag">
     Image Genomic Regions Using Fluorophore-tagged dCas9</h3>
    <p>
     Using a dCas9 fused to a fluorescent marker (such as GFP), researchers have turned dCas9 into a customizable DNA label that can be detected in live cells. By creating unique gRNAs that bind in close proximity along a stretch of genomic DNA, a technique referred to as “tiling”, researchers have imaged specific regions of the genome. The tiling technique does require multiple gRNAs to bind near one another in order to produce a detectable signal.
    </p>
    <p> <em>Browse Plasmids:</em> <a href="https://www.addgene.org/crispr/visualize/">
      Label
     </a>
    </p>
    <h3 id="genomic-purify">
     Purify Genomic Regions Using dCas9</h3>
    <p>
     Building on the well-established concept of ChIP ( <u>Ch</u> romatin <u>I</u> mmuno <u>p</u> recipitation), researchers have created enChIP ( <u>en</u> gineered DNA-binding molecule-mediated <u>ChIP</u> ) that allows for the purification of any genomic sequence specified by a particular gRNA. A catalytically inactive dCas9 fused to an epitope tag(s) can be used to purify genomic DNA bound by the gRNA.
     <a href="http://en.wikipedia.org/wiki/Chromatin_immunoprecipitation" rel="noopener noreferrer" target="_blank">
      <span class="glyphicon glyphicon-new-window">
      </span>
      Learn more about ChIP here
     </a>
     .
    </p>
    <p> <em>Browse Plasmids:</em> <a href="https://www.addgene.org/crispr/purify/">
      Purify
     </a>
    </p>
   </section>
   <section id="alt-cas9">
    <h2>Alternatives to Cas9 for CRISPR Genome Engineering</h2>
    <p>
     While <em>S. pyogenes</em> Cas9 (SpCas9) is certainly the most commonly used CRISPR endonuclease for genome engineering, it may not be the best endonuclease for every application. For example, the PAM sequence for SpCas9 (5′ NGG 3′) is abundant throughout the human genome, but a NGG sequence may not be positioned correctly to target your desired genes for modification. This is of particular concern when trying to edit a gene using
     <a href="http://blog.addgene.org/crispr-101-homology-directed-repair">
      Homology Directed Repair (HDR)
     </a>
     , which requires PAM sequences in very close proximity to the region to be edited.
     <a href="https://www.addgene.org/browse/article/14602/">
      In the Joung lab, Kleinsteiver et al.
     </a>
     has generated synthetic SpCas9-derived variants with non-NGG PAM sequences. The inclusion of these variants into the CRISPR arsenal effectively doubles the targeting range of CRISPR in the human genome.
    </p>
    <p>
     <a href="https://www.addgene.org/browse/article/15917/">
      Additional Cas9 orthologs from various species have been identified
     </a>
     and these “non-SpCas9s” bind a variety of PAM sequences. These non-SpCas9s may have other characteristics that make them more useful than SpCas9 for specific applications. For example, the relatively large size of SpCas9 (∼4kb coding sequence) means that plasmids carrying the SpCas9 cDNA cannot be efficiently packaged into
     <a href="https://www.addgene.org/viral-vectors/aav/aav-guide/">
      adeno-associated virus (AAV)
     </a>
     . Conversely, the coding sequence for <em>Staphylococcus aureus</em> Cas9 (SaCas9) is ∼1 kilobase shorter than SpCas9, allowing it to be efficiently packaged into AAV. Similar to SpCas9, the SaCas9 endonuclease is capable of modifying target genes in mammalian cells <em>in vitro</em> and in mice <em>in vivo</em> .
    </p>
    <p>
     Another limitation of SpCas9 has to do with the low efficiency of making specific genetic edits via HDR.
     <a href="https://www.addgene.org/browse/article/15925/">
      In the Zhang lab, Zetsche et al.
     </a>
     describes two RNA-guided endonucleases from the Cpf1 family that display cleavage activity in mammalian cells. Unlike Cas9 nucleases, the result of Cpf1-mediated DNA cleavage is a double-strand break with a short 3′ overhang. Cpf1’s staggered cleavage pattern opens up the possibility of directional gene transfer, analogous to traditional restriction enzyme cloning, which may increase the efficiency of gene editing. Like the Cas9 variants and orthologs described above, Cpf1 also expands the number of sites that can be targeted by CRISPR to AT-rich regions or AT-rich genomes that lack the NGG PAM sites favored by SpCas9.
    </p>
   </section>
   <hr class="shortline">
   <section class="textspace" id="plan-experiment">
    <h2>Plan Your CRISPR Experiment</h2>
    <h3>Get Started</h3>
    <p>
     CRISPR/Cas9 is a powerful system that enables researchers to manipulate the genome of target cells like never before. This section will provide a general framework to get you started using CRISPR/Cas9 in your research. Although we will focus on using CRISPR in mammalian cells, many of these principles apply to using CRISPR in other organisms. First, consider the genetic manipulation that is necessary to model your specific disease or process of interest. Do you want to:
    </p>
    <ul>
     <li>Generate complete and permanent loss of gene expression or function (i.e. knock-out)?</li>
     <li>Express a mutated version of the gene (e.g. point mutant)?</li>
     <li>Increase or decrease expression of a target gene?</li>
    </ul>
    <p>
     Once you have a clear understanding of what you are trying to do, you are ready to start navigating the different reagents that are available for your particular experiment.
    </p>
    <h3 id="select-goal">
     Select Your Desired Genetic Manipulation</h3>
    <p>
     Different genetic manipulations require different CRISPR components. Selecting a specific genetic manipulation can be a good way to narrow down which reagents are appropriate for a given experiment. Make sure to check whether reagents are available to carry out your specific genetic manipulation in your specific model organism. There may not be a perfect plasmid for your specific application, and in such a case, it may be necessary to customize an existing reagent to suit your needs.
    </p>
    <table class="table table-striped table-hover">
     <thead>
      <tr>
       <th>Genetic Manipulation</th>
       <th>Application</th>
       <th class="tbl-col-20">
        Cas9
       </th>
       <th>gRNA</th>
       <th>Additional Considerations</th>
      </tr>
     </thead>
     <tbody>
      <tr>
       <td>
        Knock-out
       </td>
       <td>
        Permanently disrupt gene function in a particular cell type or organism without regard for specific mutation
       </td>
       <td>
        Cas9 (or Cas9 nickase)
       </td>
       <td>
        Single (or dual) gRNA targeting 5′ exon or essential protein domains
       </td>
       <td>
        Dual-nickase approach increases specificity but is less efficient
       </td>
      </tr>
      <tr>
       <td>
        Edit
       </td>
       <td>
        Generate a specific user-defined sequence change in a particular gene, such as generating a point mutation or inserting a tag
       </td>
       <td>
        Cas9 (or Cas9 nickase)
       </td>
       <td>
        Single (or dual) gRNA targeting the region where the edit should be made
       </td>
       <td>
        Requires DNA template for HDR; Efficiency is reduced compared to knock-out
       </td>
      </tr>
      <tr>
       <td>
        Repress or Interfere (CRISPRi)
       </td>
       <td>
        Reduce expression of a particular gene(s) without permanently modifying the genome
       </td>
       <td>
        dCas9 or dCas9-repressor (such as dCas9-KRAB)
       </td>
       <td>
        gRNA(s) targeting promoter elements of target gene
       </td>
       <td>
        dCas9-KRAB is more effective than dCas9 alone for mammalian cell lines
       </td>
      </tr>
      <tr>
       <td>
        Activate (CRISPRa)
       </td>
       <td>
        Increase expression of an endogenous gene(s) without permanently modifying the genome
       </td>
       <td>
        dCas9-activator (such as dCas9-VP64)
       </td>
       <td>
        gRNA(s) targeting promoter elements of target gene
       </td>
       <td>
        Many different activators exist, all derived from <em>S. pyogenes</em> dCas9
       </td>
      </tr>
     </tbody>
    </table>
    <h3 id="select-model">
     Select Expression System</h3>
    <p>
     To use the CRISPR system, you will need both gRNA and Cas9 expressed in your target cells. The expression system you need will depend upon your specific application. For example, certain cell types are easier to transfect than others. For easy-to-transfect cell types (e.g. HEK293 cells), transfection with standard transfection reagents may be sufficient to express both your gRNA and Cas9. For more difficult cells (e.g. primary cells), viral delivery of CRISPR reagents may be more appropriate.
    </p>
    <p>
     The table below summarizes the major expression systems and variables for using CRISPR in mammalian cells. Some of the variables include:
    </p>
    <ul>
     <li>Species of Cas9 and gRNA</li>
     <li>Species of promoter and expression pattern of promoter for Cas9 and gRNA</li>
     <li>Presence of a selectable marker (drug or fluorophore)</li>
     <li>Delivery method</li>
    </ul>
    <table class="table table-striped table-hover">
     <thead>
      <tr>
       <th class="tbl-col-20">
        Expression System
       </th>
       <th>Components of System</th>
       <th>Application</th>
      </tr>
     </thead>
     <tbody>
      <tr>
       <td>
        Mammalian expression vector
       </td>
       <td>
        <ul class="table-list">
         <li>Cas9 promoter can be constitutive (CMV, EF1alpha, CBh) or inducible (Tet-ON); U6 promoter is typically used for gRNA</li>
         <li>May contain reporter gene (e.g. GFP) to identify and enrich positive cells, or selection marker to generate stable cell lines (for details,<a href="http://blog.addgene.org/plasmids-101-mammalian-vectors">see Addgene’s Plasmids 101: Mammalian vectors</a>)</li>
        </ul>
       </td>
       <td>
        Transient or stable expression of Cas9 and/or gRNA in a mammalian cell line that can be transfected at high efficiency
       </td>
      </tr>
      <tr>
       <td>
        Lentiviral transduction
       </td>
       <td>
        <ul class="table-list">
         <li>Cas9 and gRNA can be present in a single lentiviral transfer vector or separate transfer vectors</li>
         <li>May contain reporter gene (e.g. GFP) or selection marker to identify and enrich positive cells</li>
         <li>Packaging and envelope plasmids provide the necessary components to make lentiviral particles (for details about lentivirus,<a href="https://www.addgene.org/viral-vectors/lentivirus/lenti-guide/">see Addgene’s Lentivirus guide</a>)</li>
        </ul>
       </td>
       <td>
        <ul class="table-list">
         <li>Stable, tunable expression of Cas9 and/or gRNA in a wide variety of mammalian cell lines</li>
         <li>Useful for difficult to transfect cell types and can be used <em>in vivo</em> </li>
         <li>A common choice for conducting genome-wide screens using CRISPR/Cas9</li>
        </ul>
       </td>
      </tr>
      <tr>
       <td>
        AAV transduction
       </td>
       <td>
        <ul class="table-list">
         <li>Only compatible with SaCas9 (packaging limit ∼4.5kb)</li>
         <li>CRISPR elements are inserted into an AAV transfer vector and used to generate AAV particles (<a href="https://www.addgene.org/viral-vectors/aav/aav-guide/">for details, see Addgene’s AAV guide</a>)</li>
        </ul>
       </td>
       <td>
        <ul class="table-list">
         <li>Transient or stable expression of SaCas9 and/or gRNA</li>
         <li>Infects dividing and non-dividing cells</li>
         <li>AAV is least toxic method for <em>in vivo</em> viral delivery</li>
        </ul>
       </td>
      </tr>
      <tr>
       <td>
        Cas9 mRNA and gRNA
       </td>
       <td>
        Plasmids containing gRNA and Cas9 are used in <em>in vitro</em> transcription reactions to generate mature Cas9 mRNA and gRNA, then delivered to target cells (i.e. microinjection or electroporation)
       </td>
       <td>
        <ul class="table-list">
         <li>Transient expression of CRISPR components</li>
         <li>Expression decreases as RNA is degraded within the cell</li>
         <li>Can be used for generating transgenic embryos</li>
        </ul>
       </td>
      </tr>
      <tr>
       <td>
        Cas9-gRNA riboprotein complexes
       </td>
       <td>
        Purified Cas9 protein and <em>in vitro</em> transcribed gRNA are combined to form a Cas9-gRNA complex and delivered to cells using cationic lipids
       </td>
       <td>
        <ul class="table-list">
         <li>Transient expression of CRISPR components</li>
         <li>Expression decreases as gRNA and Cas9 protein are degraded within the cell</li>
        </ul>
       </td>
      </tr>
     </tbody>
    </table>
    <h4>Additional Addgene Resources:</h4>
    <ul style="margin-bottom: 2em;">
     <li><a href="https://www.addgene.org/crispr/reference/#protocols">Depositor protocols</a>for plasmids at Addgene</li>
     <li>JOVE Video Reprint:<a href="https://www.addgene.org/crispr/genomic-deletions-mammalian-cell-lines/">Protocol for genomic deletions in mammalian cell lines</a></li>
    </ul>
    <h3 id="design-grna">
     Select Your Target Sequence and Design Your gRNA</h3>
    <p>
     Once you have selected your CRISPR components and method of delivery, you are ready to select a target sequence and design your gRNA. A general overview of how to design a gRNA is presented below.
    </p>
    <div>
     <div class="pull-right" id="grna-design-img">
      <img class="filer_image " alt="grna design process diagram" src="./Addgene_ CRISPR_Cas9 Guide_files/grna-design.png__386x500_q85_subsampling-2_upscale.png" height="500">

     </div>
     <ol id="grna-design-list">
      <li id="know-cells">
       Know your cell line and genome sequence
       <span class="ol-unbolding">
        <p>
         The cell line you choose determines a variety of factors related to your experiment. The genomic sequence used to design gRNAs will depend upon the gene in question and the species from which your cells were derived. When possible, you should sequence the region you are planning to modify prior to designing your gRNA, as sequence variation between your gRNA targeting sequence and target DNA may result in reduced cleavage. The number of alleles for each gene may vary depending on the specific cell line or organism, which may affect the observed efficiency of knock-out or knock-in using CRISPR/Cas9.
        </p>
       </span>
      </li>
      <li id="select-gene">
       Select gene and genetic element to be manipulated
       <span class="ol-unbolding">
        <p>
         In order to manipulate a given gene using CRISPR/Cas9, you will have to identify the genomic sequence for the gene you are trying to target. However, the exact region of the gene you target will depend on your specific application. For example:
        </p>
        <ul class="spacing-lists">
         <li>To activate or repress a target gene using dCas9-activators or dCas9-repressors, gRNAs should be targeted to the promoter driving expression of your gene of interest.</li>
         <li>For genetic knock-outs, gRNAs commonly target 5′ constitutively expressed exons, which reduces the chances that the targeted region is removed from the mRNA due to alternative splicing. Exons near the N-terminus are targeted since frameshift mutations here will increase the likelihood that a nonfunctional protein product is produced.</li>
         <li>Alternatively, gRNAs can be designed to target exons that code for known essential protein domains. The benefit of this approach is that non-frameshift mutations such as insertions or deletions (which commonly result from DSBs) are more likely to alter protein function when they occur in protein domains that are essential for protein function.</li>
         <li>For gene editing experiments using HDR, it is essential that the target sequence be close to the location of the desired edit. In this case, it is necessary to identify the exact location where the edit should occur and select a target sequence nearby.</li>
        </ul>
       </span>
      </li>
      <li id="select-grna">
       Select gRNAs based on predicted “on-target” and “off-target” activity
       <span class="ol-unbolding">
        <p>
         A PAM sequence is absolutely necessary for Cas9 to bind target DNA. As such, one must identify all PAM sequences within the genetic region to be targeted (PAM is 5′ NGG 3′ for SpCas9). If there are no PAM sequences within your desired sequence, you may want to consider using a Cas9 from a different species or a <em>S. pyogenes</em> Cas9 variant that binds a PAM sequence that is present in the genomic region you want to target (
         <a href="https://www.addgene.org/crispr/guide/#pam-table">
          see additional Cas9 variants and PAM sequences
         </a>
         ). Once all possible PAM sequences and putative target sites have been identified, it is time to choose which site is likely to result in the most efficient on-target cleavage.
        </p>
        <p>
         Clearly, the gRNA targeting sequence needs to match the target sequence, but it is also critical to ensure that the gRNA targeting sequence does NOT match additional sites within the genome. In a perfect world, your gRNA targeting sequence would have perfect homology to your target with no homology elsewhere in the genome. Realistically, a given gRNA targeting sequence will have additional sites throughout the genome where partial homology exists. These sites are called “off-targets” and should be considered when designing a gRNA for your experiments. In general, off-target sites are not cleaved as efficiently when mismatches occur near the PAM sequence, so gRNAs with no homology or those with mismatches close to the PAM sequence will have the highest specificity.
        </p>
        <p>
         In addition to “off-target activity”, it is also important to consider factors that maximize cleavage of the desired target sequence (“on-target activity”). It is now understood that two gRNA targeting sequences, each having 100% homology to the target DNA may not result in equivalent cleavage efficiency. In fact, cleavage efficiency may increase or decrease depending upon the specific nucleotides within the selected target sequence. For example, gRNA targeting sequences containing a G nucleotide at position 20 (1 bp upstream of the PAM) may be more efficacious than gRNAs containing a C nucleotide at the same position in spite of being a perfect match for the target sequence. Therefore, close examination of predicted on-target and off-target activity of each potential gRNA targeting sequence is necessary to design the best gRNA for your experiment.
        </p>
        <p>
         Several gRNA design programs have been developed that are capable of locating potential PAM and target sequences and ranking the associated gRNAs based on their predicted on-target and off-target activity (
         <a href="https://www.addgene.org/crispr/reference/">
          see gRNA design software
         </a>
         ). Additionally, many plasmids containing “validated” gRNAs are now available through Addgene. These plasmids contain gRNAs that have been used successfully in genome engineering experiments and have been published in peer-reviewed journals. Using validated gRNAs can save your lab valuable time and resources when carrying out experiments using CRISPR/Cas9.
        </p>
        <p> <em>Browse Plasmids:</em> <a href="https://www.addgene.org/crispr/validated-grnas/">
          Validated gRNAs
         </a>
        </p>
       </span>
      </li>
      <li id="clone-grna">
       Synthesize and clone desired gRNAs
       <span class="ol-unbolding">
        <p>
         Once you have selected your target sequences it is time to design your gRNA oligos and clone these oligos into your desired vector. In many cases, targeting oligos are synthesized, annealed, and inserted into plasmids containing the gRNA scaffold using standard restriction-ligation cloning. However, the exact cloning strategy will depend on the gRNA vector you have chosen, so it is best to review the protocol associated with the specific plasmid in question (
         <a href="https://www.addgene.org/crispr/reference/#protocols">
          see CRISPR protocols from Addgene depositors
         </a>
         ).
        </p>
       </span>
      </li>
      <li id="reagent-delivery">
       Deliver Cas9 and gRNA
       <span class="ol-unbolding">
        <p>
         Your method of delivery will be determined by the specific expression system that you have chosen. It may be necessary to optimize a protocol for delivery of gRNA and Cas9 to your target cells, as transfection efficiency will vary based on the method of delivery and the cell type tested.
         <a href="http://blog.addgene.org/crispr-101-mammalian-expression-systems-and-delivery-methods">
          Read our blog post for more information about delivery in mammalian systems.
         </a>
        </p>
       </span>
      </li>
      <li id="validate-results">
       Validate genetic modification
       <span class="ol-unbolding">
        <p>
         Once you have successfully delivered the gRNA and Cas9 to your target cells, it is time to validate your genome edit. Delivery of Cas9 and gRNA to wild-type cells will result in several possible genotypes within the resulting “mutant” cell population. A percentage of cells may be wild-type due to either (1) lack of gRNA and/or Cas9 expression, or (2) lack of efficient target cleavage in cells expressing both Cas9 and gRNA. Cells that have undergone modification (mutant cells) can be homozygous (modification of both alleles) or heterozygous (modification of a single allele). Furthermore, in mutant cells containing two mutated alleles, each mutated allele may be different owing to the error-prone nature of NHEJ. Even for gene editing experiments using HDR, not every mutated allele will contain the desired edit as a large percentage of DSBs are still repaired by NHEJ. Therefore, the end result of most experiments is a heterogeneous population of cells containing a wide variety of mutations or edits within target genes.
        </p>
        <p>
         How do you determine that your desired edit has occurred? The exact method necessary to validate your edit will depend upon your specific application, and in some cases, new approaches must be devised. However, there are several common ways to verify that your cells contain your desired edit, including but not limited to:
        </p>
        <ol id="edit-validate">
         <li>Mismatch-cleavage assay (for NHEJ repaired DSBs): Provides a semi-quantitative readout of the percentage of alleles that have been mutated within a mixed cell population. The region of interest is PCR amplified, PCR products are denatured-renatured, treated with a nuclease that cleaves DNA heteroduplexes, and run on an agarose gel to identify DNA fragments.</li>
         <li>PCR and restriction digest (for HDR repaired DSBs): For small nucleotide edits that introduce a novel restriction site. The region of interest is PCR amplified, digested with the appropriate restriction enzyme and run on an agarose gel to identify DNA fragments.</li>
         <li>PCR amplification and gel electrophoresis (for HDR or NHEJ): For large deletions or insertions, the region of interest can be PCR amplified using primers that (A) flank the region of interest (deletions or small insertions) or (B) span the genome-insert boundary (insertions only). The PCR product is then run on an agarose gel to determine whether the edit was successful.</li>
         <li>PCR amplification, subcloning and Sanger sequencing (for HDR or NHEJ): Provides semi-quantitative assessment of targeting frequency and exact sequence of targeted alleles. Involves PCR amplification of targeted region from DNA, subcloning into a plasmid, and screening individual clones.</li>
         <li>PCR amplification and next-generation sequencing (for HDR or NHEJ): Provides quantitative assessment of the genome edits in your target sequence and can also be used to examine off-target effects.</li>
        </ol>
        <p style="margin-top: 1.5em;">
         More information on each of these techniques can be found on our blog post,
         <a href="http://blog.addgene.org/crispr-101-validating-your-genome-edit">
          CRISPR 101: Validating your Genome Edit.
         </a>
        </p>
       </span>
      </li>
     </ol>
    </div>
   </section>
   <hr class="shortline">
   <section class="textspace">
    <h2>Resources</h2>
    <h3 id="web-references">
     Addgene Website &amp; Blog References</h3>
    <h4><em>Website</em></h4>
    <ul>
     <li><a href="https://www.addgene.org/crispr/">CRISPR plasmids at Addgene</a></li>
     <li><a href="https://www.addgene.org/crispr/reference/">CRISPR software and depositor protocols</a></li>
     <li>JOVE video reprint:<a href="https://www.addgene.org/crispr/genomic-deletions-mammalian-cell-lines/">Protocol for genomic deletions in mammalian cell lines</a></li>
     <li><a href="https://www.addgene.org/viral-vectors/">Viral vectors at Addgene</a></li>
    </ul>
    <h4><em>Blog</em></h4>
    <ul>
     <li><a href="http://blog.addgene.org/the-crispr-software-matchmaker-a-new-tool-for-choosing-the-best-crispr-software-for-your-needs">CRISPR software guide</a></li>
     <li><a href="http://blog.addgene.org/genome-wide-screening-using-crispr/cas9">Genome-wide screening guide</a></li>
     <li><a href="http://blog.addgene.org/crispr-101-validating-your-genome-edit">Validation of your genome edits</a></li>
     <li><a href="http://blog.addgene.org/crispr-101-homology-directed-repair">HDR (Homology directed repair) guide</a></li>
    </ul>
    <h3 id="pam-table">
     Cas9 Species/Variants and PAM Sequences</h3>
    <table class="table table-striped table-hover table-condensed" style="width: 100%;">
     <thead>
      <tr>
       <th>Species/Variant of Cas9</th>
       <th>PAM Sequence</th>
      </tr>
     </thead>
     <tbody id="pam-table">
      <tr>
       <td>
        Streptococcus pyogenes (SP); SpCas9
       </td>
       <td>
        NGG
       </td>
      </tr>
      <tr>
       <td>
        SpCas9 D1135E variant
       </td>
       <td>
        NGG (reduced NAG binding)
       </td>
      </tr>
      <tr>
       <td>
        SpCas9 VRER variant
       </td>
       <td>
        NGCG
       </td>
      </tr>
      <tr>
       <td>
        SpCas9 EQR variant
       </td>
       <td>
        NGAG
       </td>
      </tr>
      <tr>
       <td>
        SpCas9 VQR variant
       </td>
       <td>
        NGAN or NGNG
       </td>
      </tr>
      <tr>
       <td>
        Staphylococcus aureus (SA); SaCas9
       </td>
       <td>
        NNGRRT or NNGRR(N)
       </td>
      </tr>
      <tr>
       <td>
        Neisseria meningitidis (NM)
       </td>
       <td>
        NNNNGATT
       </td>
      </tr>
      <tr>
       <td>
        Streptococcus thermophilus (ST)
       </td>
       <td>
        NNAGAAW
       </td>
      </tr>
      <tr>
       <td>
        Treponema denticola (TD)
       </td>
       <td>
        NAAAAC
       </td>
      </tr>
      <tr>
       <td>
        Additional Cas9s from various species
       </td>
       <td>
        PAM sequence may not be characterized
       </td>
      </tr>
     </tbody>
    </table>
    <p>
     The majority of the
     <a href="https://www.addgene.org/crispr/">
      CRISPR plasmids
     </a>
     in Addgene’s collection are from <em>S. pyogenes</em> unless otherwise noted.
    </p>
    <!-- separate glossary HTML plugin -->
    



<h3 id="glossary">Glossary</h3>
  
  <table class="table table-striped table-hover" style="witdh: 80%;">
    <thead>
        <tr>
            <th>Term</th>
            <th>Definition</th>
        </tr>
    </thead>
    
    <tbody>
        <tr>
            <td>Cas</td>
            <td><u>C</u>RISPR <u>As</u>sociated Protein, the Cas9 nuclease is the active enzyme for the Type II CRISPR system</td>
        </tr>
        
        <tr>
            <td>CRISPR</td>
            <td><u>C</u>lustered <u>R</u>egularly <u>I</u>nterspaced <u>S</u>hort <u>P</u>alendromic <u>R</u>epeat, a region in bacterial genomes used in pathogen defense</td>
        </tr>    
            
        <tr>   
            <td>CRISPRa</td>
            <td>CRISPR Activation, using a dCas9 or dCas9-activator with a gRNA to increase transcription of a target gene</td>
        </tr>    
        
        <tr>
            <td>CRISPRi</td>
            <td>CRISPR Interference, using a dCas9 or dCas9-repressor with a gRNA to repress/decrease transcription of a target gene</td>
        </tr>
        
        <tr>
            <td>Cut</td>
            <td>A DNA double strand break, the wild type function of Cas9</td>
        </tr>
        
        <tr>
            <td>dCas9</td>
            <td>Nuclease dead Cas9, an enzymatically inactive form of Cas9; Can bind, but cannot cleave DNA</td>
        </tr>    
    
        <tr>    
            <td>DSB</td>
            <td><u>D</u>ouble <u>S</u>trand <u>B</u>reak, a break in both strands of DNA, through the use of Cas9 or two Cas9-nickases targeting opposite strands</td>
        </tr>    
    
        <tr>
            <td>Dual Nickase/Double Nick</td>
            <td>A method to decrease off-target effects by using a single Cas9 nickase and 2 different gRNAs, which bind in close proximity on opposite strands of the DNA, to create a DSB</td>
        </tr>    
    
        <tr>
            <td>enChIP</td>
            <td><u>En</u>gineered DNA-binding molecule-mediated <u>ChIP</u>, using a tagged-dCas9+gRNA to purify specific genomic regions to identify molecules associated with the genomic regions</td>
        </tr>
        
        <tr>
            <td>Genetic modification or manipulation</td>
            <td>Any genetic perturbation, including genetic knock-out, gene activation, or gene repression</td>
        </tr>    
    
        <tr>    
            <td>gRNA</td>
            <td><u>G</u>uide <u>RNA</u>, a synthetic fusion of the endogenous bacterial crRNA and tracrRNA; Provides both targeting specificity and scaffolding/binding ability for Cas9 nuclease; Does not exist in nature; Also referred to as “single guide RNA” or “sgRNA”</td>
        </tr>
        
        <tr>
            <td>gRNA scaffold sequence</td>
            <td>The sequence within the gRNA that is responsible for Cas9 binding; Does not include the 20bp spacer/targeting sequence that is used to guide Cas9 to target DNA</td>
        </tr>    
    
        <tr>    
            <td>gRNA targeting sequence</td>
            <td>The 20 nucleotides that precede the PAM sequence in the genomic DNA; What gets put into a gRNA expression plasmid; Does <strong>NOT</strong> include the PAM sequence or the gRNA scaffold sequence</td>
        </tr>    
    
        <tr>
            <td>HDR</td>
            <td><u>H</u>omology <u>D</u>irected <u>R</u>epair, a DNA repair mechanism that uses a template to repair nicks or DSBs</td>
        </tr>    
    
        <tr>
            <td>InDel</td>
            <td><u>In</u>sertion/<u>Del</u>etion, a type of mutation that can result in the disruption of a gene by shifting the ORF and/or creating premature stop codons</td>
        </tr>
        
        <tr>
            <td>NHEJ</td>
            <td><u>N</u>on-<u>H</u>omologous <u>E</u>nd-<u>J</u>oining; A DNA repair mechanism that often introduces InDels</td>
        </tr>    
    
        <tr>
            <td>Nick</td>
            <td>A break in only one strand of a double stranded DNA; Normally repaired by HDR</td>
        </tr>    
    
        <tr>
            <td>Nickase</td>
            <td>Cas9 that has one of the two nuclease domains inactivated; Can be either the RuvC or HNH domain; Capable of cleaving only one strand of target DNA</td>
        </tr>
        
        <tr>
            <td>Off-target effects or off-target activity</td>
            <td>Cas9 cleavage at undesired locations due to gRNA targeting sequence with sufficient homology to recruit Cas9 to unintended genomic locations; Can be minimized by double-nick</td>
        </tr>    
    
        <tr>
            <td>On-target activity</td>
            <td>Cas9 cleavage at a desired location due to gRNA targeting sequence with sufficient homology to recruit Cas9 to desired genomic locations</td>
        </tr>
        
        <tr>
            <td>ORF</td>
            <td><u>O</u>pen <u>R</u>eading <u>F</u>rame; The codons that make up a gene</td>
        </tr>    
    
        <tr>
            <td>PAM</td>
            <td><u>P</u>rotospacer <u>A</u>djacent <u>M</u>otif; Necessary for Cas9 to bind target DNA; Must immediately follow the target sequence</td>
        </tr>    
    
        <tr>
            <td>PCR</td>
            <td><u>P</u>olymerase <u>C</u>hain <u>R</u>eaction; Used to amplify a specific sequence of DNA</td>
        </tr>
        
        <tr>
            <td>sgRNA</td>
            <td><u>S</u>ingle <u>g</u>uide <u>RNA</u>; A synthetic RNA composed of a targeting sequence and scaffold sequence derived from endogenous bacterial crRNA and tracrRNA; Used to target Cas9 to a specific genomic locus in genome engineering experiments; Also referred to as a “gRNA”</td>
        </tr>    
    
        <tr>    
            <td>Target sequence</td>
            <td>Genomic target of the gRNA targeting sequence; The 20 nucleotides that are incorporated into the gRNA plus the PAM sequence; Not to be confused with the gRNA targeting sequence</td>
        </tr>
    </tbody>
      
  </table>




    <!-- separate publications HTML plugin -->
    



<!-- This file was automatically modified on 2017-01-03 -->
 
  <h3 id="publications">
   Publications</h3>
  <h4>2016</h4>
  <ul>
   <li><p><cite>Practical Considerations for Using Pooled Lentiviral CRISPR Libraries. McDade JR, Waxmonsky NC, Swanson LE, Fan M. <em>Curr Protoc Mol Biol.</em> 2016 Jul 1;115:31.5.1-31.5.13. doi: 10.1002/cpmb.8.<a href="https://www.ncbi.nlm.nih.gov/pubmed/27366891" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
  </ul>
  <h4>2015</h4>
  <ul>
   <li><p><cite>Cpf1 Is a Single RNA-Guided Endonuclease of a Class 2 CRISPR-Cas System. Zetsche B, Gootenberg JS, Abudayyeh OO, Slaymaker IM, Makarova KS, Essletzbichler P, Volz SE, Joung J, van der Oost J, Regev A, Koonin EV, Zhang F. <em>Cell.</em> 2015 Oct 22;163(3):759-71. doi: 10.1016/j.cell.2015.09.038.<a href="https://www.ncbi.nlm.nih.gov/pubmed/26422227" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Engineered CRISPR-Cas9 nucleases with altered PAM specificities. Kleinstiver BP, Prew MS, Tsai SQ, Topkar VV, Nguyen NT, Zheng Z, Gonzales AP, Li Z, Peterson RT, Yeh JR, Aryee MJ, Joung JK. <em>Nature.</em> 2015 Jul 23;523(7561):481-5. doi: 10.1038/nature14592.<a href="https://www.ncbi.nlm.nih.gov/pubmed/26098369" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>CRISPR-Cas9-Mediated Genetic Screening in Mice with Haploid Embryonic Stem Cells Carrying a Guide RNA Library. Zhong C, Yin Q, Xie Z, Bai M, Dong R, Tang W, Xing YH, Zhang H, Yang S, Chen LL, Bartolomei MS, Ferguson-Smith A, Li D, Yang L, Wu Y, Li J. <em>Cell Stem Cell.</em> 2015 Aug 6;17(2):221-32. doi: 10.1016/j.stem.2015.06.005.<a href="https://www.ncbi.nlm.nih.gov/pubmed/26165924" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Multiplexable, locus-specific targeting of long RNAs with CRISPR-Display. Shechner DM, Hacisuleyman E, Younger ST, Rinn JL. <em>Nat Methods.</em> 2015 Jul;12(7):664-70. doi: 10.1038/nmeth.3433.<a href="https://www.ncbi.nlm.nih.gov/pubmed/26030444" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Discovery of cancer drug targets by CRISPR-Cas9 screening of protein domains. Shi J, Wang E, Milazzo JP, Wang Z, Kinney JB, Vakoc CR. <em>Nat Biotechnol.</em> 2015 Jun;33(6):661-7. doi: 10.1038/nbt.3235.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25961408" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Electroporation enables the efficient mRNA delivery into the mouse zygotes and facilitates CRISPR/Cas9-based genome editing. Hashimoto M, Takemoto T. <em>Sci Rep.</em> 2015 Jun 11;5:11315. doi: 10.1038/srep11315.<a href="https://www.ncbi.nlm.nih.gov/pubmed/26066060" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>High-throughput functional genomics using CRISPR-Cas9. Shalem O, Sanjana NE, Zhang F. <em>Nat Rev Genetics.</em> 2015 May;16(5):299-311. doi: 10.1038/nrg3899.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25854182" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Increasing the efficiency of homology-directed repair for CRISPR-Cas9-induced precise gene editing in mammalian cells. Chu VT, Weber T, Wefers B, Wurst W, Sander S, Rajewsky K, Kühn R. <em>Nat Biotechnol.</em> 2015 May;33(5):543-8. doi: 10.1038/nbt.3198.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25803306" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Highly efficient Cas9-mediated transcriptional programming. Chavez A, Scheiman J, Vora S, Pruitt BW, Tuttle M, P R Iyer E, Lin S, Kiani S, Guzman CD, Wiegand DJ, Ter-Ovanesyan D, Braff JL, Davidsohn N, Housden BE, Perrimon N, Weiss R, Aach J, Collins JJ, Church GM. <em>Nat Methods.</em> 2015 Apr;12(4):326-8. doi: 10.1038/nmeth.3312.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25730490" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>In vivo genome editing using Staphylococcus aureus Cas9. Ran FA, Cong L, Yan WX, Scott DA, Gootenberg JS, Kriz AJ, Zetsche B, Shalem O, Wu X, Makarova KS, Koonin EV, Sharp PA, Zhang F. <em>Nature.</em> 2015 Apr 9;520(7546):186-91. doi: 10.1038/nature14299.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25830891" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Genome engineering using CRISPR-Cas9 system. Cong L, Zhang F. <em>Methods Mol Biol.</em> 2015;1239:197-217. doi: 10.1007/978-1-4939-1862-1_10.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25408407" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Genome-scale transcriptional activation by an engineered CRISPR-Cas9 complex. Konermann S, Brigham MD, Trevino AE, Joung J, Abudayyeh OO, Barcena C, Hsu PD, Habib N, Gootenberg JS, Nishimasu H, Nureki O, Zhang F. <em>Nature.</em> 2015 Jan 29;517(7536):583-8. doi: 10.1038/nature14136.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25494202" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Engineering complex synthetic transcriptional programs with CRISPR RNA scaffolds. Zalatan JG, Lee ME, Almeida R, Gilbert LA, Whitehead EH, La Russa M, Tsai JC, Weissman JS, Dueber JE, Qi LS, Lim WA. <em>Cell.</em> 2015 Jan 15;160(1-2):339-50. doi: 10.1016/j.cell.2014.11.052.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25533786" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Cationic lipid-mediated delivery of proteins enables efficient protein-based genome editing in vitro and in vivo. Zuris JA, Thompson DB, Shu Y, Guilinger JP, Bessen JL, Hu JH, Maeder ML, Joung JK, Chen ZY, Liu DR. <em>Nat Biotechnol.</em> 2015 Jan;33(1):73-80. doi: 10.1038/nbt.3081.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25357182" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
  </ul>
  <h4>2014</h4>
  <ul>
   <li><p><cite>Enhanced homology-directed human genome engineering by controlled timing of CRISPR/Cas9 delivery. Lin S, Staahl BT, Alla RK, Doudna JA. <em>Elife.</em> 2014 Dec 15;3:e04766. doi: 10.7554/eLife.04766.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25497837" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>In vivo engineering of oncogenic chromosomal rearrangements with the CRISPR/Cas9 system. Maddalo D, Manchado E, Concepcion CP, Bonetti C, Vidigal JA, Han YC, Ogrodowski P, Crippa A, Rekhtman N, de Stanchina E, Lowe SW, Ventura A. <em>Nature.</em> 2014 Dec 18;516(7531):423-7. doi: 10.1038/nature13902.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25337876" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Rational design of highly active sgRNAs for CRISPR-Cas9-mediated gene inactivation. Doench JG, Hartenian E, Graham DB, Tothova Z, Hegde M, Smith I, Sullender M, Ebert BL, Xavier RJ, Root DE. <em>Nat Biotechnol.</em> 2014 Dec;32(12):1262-7. doi: 10.1038/nbt.3026.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25184501" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Genome editing using Cas9 nickases. Trevino AE, Zhang F. <em>Methods Enzymol.</em> 2014;546:161-74. doi: 10.1016/B978-0-12-801185-0.00008-8.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25398340" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Multiplex CRISPR/Cas9-based genome engineering from a single lentiviral vector. Kabadi AM, Ousterout DG, Hilton IB, Gersbach CA. <em>Nucleic Acids Res.</em> 2014 Oct 29;42(19):e147. doi: 10.1093/nar/gku749.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25122746" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>A protein-tagging system for signal amplification in gene expression and fluorescence imaging. Tanenbaum ME, Gilbert LA, Qi LS, Weissman JS, Vale RD. <em>Cell.</em> 2014 Oct 23;159(3):635-46. doi: 10.1016/j.cell.2014.09.039.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25307933" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Structural basis of PAM-dependent target DNA recognition by the Cas9 endonuclease. Anders C, Niewoehner O, Duerst A, Jinek M. <em>Nature.</em> 2014 Sep 25;513(7519):569-73. doi: 10.1038/nature13579.<a href="https://www.ncbi.nlm.nih.gov/pubmed/25079318" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Multiplex genome engineering in human cells using all-in-one CRISPR/Cas9 vector system. Sakuma T, Nishikawa A, Kume S, Chayama K, Yamamoto T. <em>Sci Rep.</em> 2014 Jun 23;4:5400. doi: 10.1038/srep05400.<a href="https://www.ncbi.nlm.nih.gov/pubmed/24954249" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Crystal structure of Cas9 in complex with guide RNA and target DNA. Nishimasu H, Ran FA, Hsu PD, Konermann S, Shehata SI, Dohmae N, Ishitani R, Zhang F, Nureki O. <em>Cell.</em> 2014 Feb 27;156(5):935-49. doi: 10.1016/j.cell.2014.02.001.<a href="https://www.ncbi.nlm.nih.gov/pubmed/24529477" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Genetic screens in human cells using the CRISPR-Cas9 system. Wang T, Wei JJ, Sabatini DM, Lander ES. <em>Science.</em> 2014 Jan 3;343(6166):80-4. doi: 10.1126/science.1246981.<a href="https://www.ncbi.nlm.nih.gov/pubmed/24336569" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Genome-scale CRISPR-Cas9 knockout screening in human cells. Shalem O, Sanjana NE, Hartenian E, Shi X, Scott DA, Mikkelsen TS, Heckl D, Ebert BL, Root DE, Doench JG, Zhang F. <em>Science.</em> 2014 Jan 3;343(6166):84-7. doi: 10.1126/science.1247005.<a href="https://www.ncbi.nlm.nih.gov/pubmed/24336571" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
  </ul>
  <h4>2013</h4>
  <ul>
   <li><p><cite>Genome engineering using the CRISPR-Cas9 system. Ran FA, Hsu PD, Wright J, Agarwala V, Scott DA, Zhang F. <em>Nat Protoc.</em> 2013 Nov;8(11):2281-308. doi: 10.1038/nprot.2013.143.<a href="https://www.ncbi.nlm.nih.gov/pubmed/24157548" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Multiplexed activation of endogenous genes by CRISPR-on, an RNA-guided transcriptional activator system. Cheng AW, Wang H, Yang H, Shi L, Katz Y, Theunissen TW, Rangarajan S, Shivalila CS, Dadon DB, Jaenisch R. <em>Cell Res.</em> 2013 Oct;23(10):1163-71. doi: 10.1038/cr.2013.122.<a href="https://www.ncbi.nlm.nih.gov/pubmed/23979020" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>CAS9 transcriptional activators for target specificity screening and paired nickases for cooperative genome engineering. Mali P, Aach J, Stranges PB, Esvelt KM, Moosburner M, Kosuri S, Yang L, Church GM. <em>Nat Biotechnol.</em> 2013 Sep;31(9):833-8. doi: 10.1038/nbt.2675.<a href="https://www.ncbi.nlm.nih.gov/pubmed/23907171" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>DNA targeting specificity of RNA-guided Cas9 nucleases. Hsu PD, Scott DA, Weinstein JA, Ran FA, Konermann S, Agarwala V, Li Y, Fine EJ, Wu X, Shalem O, Cradick TJ, Marraffini LA, Bao G, Zhang F. <em>Nat Biotechnol.</em> 2013 Sep;31(9):827-32. doi: 10.1038/nbt.2647.<a href="https://www.ncbi.nlm.nih.gov/pubmed/23873081" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>High-frequency off-target mutagenesis induced by CRISPR-Cas nucleases in human cells. Fu Y, Foden JA, Khayter C, Maeder ML, Reyon D, Joung JK, Sander JD. <em>Nat Biotechnol.</em> 2013 Sep;31(9):822-6. doi: 10.1038/nbt.2623.<a href="https://www.ncbi.nlm.nih.gov/pubmed/23792628" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes. Gilbert LA, Larson MH, Morsut L, Liu Z, Brar GA, Torres SE, Stern-Ginossar N, Brandman O, Whitehead EH, Doudna JA, Lim WA, Weissman JS, Qi LS. <em>Cell.</em> 2013 Jul 18;154(2):442-51. doi: 10.1016/j.cell.2013.06.044.<a href="https://www.ncbi.nlm.nih.gov/pubmed/23849981" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>One-step generation of mice carrying mutations in multiple genes by CRISPR/Cas-mediated genome engineering. Wang H, Yang H, Shivalila CS, Dawlaty MM, Cheng AW, Zhang F, Jaenisch R. <em>Cell.</em> 2013 May 9;153(4):910-8. doi: 10.1016/j.cell.2013.04.025.<a href="https://www.ncbi.nlm.nih.gov/pubmed/23643243" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression. Qi LS, Larson MH, Gilbert LA, Doudna JA, Weissman JS, Arkin AP, Lim WA. <em>Cell.</em> 2013 Feb 28;152(5):1173-83. doi: 10.1016/j.cell.2013.02.022.<a href="https://www.ncbi.nlm.nih.gov/pubmed/23452860" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>Multiplex genome engineering using CRISPR/Cas systems. Cong L, Ran FA, Cox D, Lin S, Barretto R, Habib N, Hsu PD, Wu X, Jiang W, Marraffini LA, Zhang F. <em>Science.</em> 2013 Feb 15;339(6121):819-23. doi: 10.1126/science.1231143.<a href="https://www.ncbi.nlm.nih.gov/pubmed/23287718" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
   <li><p><cite>RNA-guided human genome engineering via Cas9. Mali P, Yang L, Esvelt KM, Aach J, Guell M, DiCarlo JE, Norville JE, Church GM. <em>Science.</em> 2013 Feb 15;339(6121):823-6. doi: 10.1126/science.1232033.<a href="https://www.ncbi.nlm.nih.gov/pubmed/23287722" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
  </ul>
  <h4>Pre-2013</h4>
  <ul>
   <li><p><cite>Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, Paulovich A, Pomeroy SL, Golub TR, Lander ES, Mesirov JP. <em>Proc Natl Acad Sci U S A.</em> 2005 Oct 25;102(43):15545-50.<a href="https://www.ncbi.nlm.nih.gov/pubmed/16199517" rel="noopener noreferrer" target="_blank">PubMed</a>.</cite></p></li>
  </ul>
 




   </section>
  </div>
 







            </div>
            
        </main>

        </div> <!-- close #page-body -->

        <footer>
<div id="addgene-footer">

  <div id="addgene-social" class="container">
    <div class="row">

      <div id="social-icons" class="text-right">
        <a href="http://blog.addgene.org/" target="_blank" rel="noopener noreferrer">
          <img src="./Addgene_ CRISPR_Cas9 Guide_files/Feed-icon.svg" alt="Addgene Blog" width="30" height="32">
        </a>
        <a href="https://www.facebook.com/addgene" target="_blank" rel="noopener noreferrer">
          <img src="./Addgene_ CRISPR_Cas9 Guide_files/facebook-icon.png" alt="Addgene Facebook Page" width="30" height="32">
        </a>
        <a href="https://www.twitter.com/addgene/" target="_blank" rel="noopener noreferrer">
          <img src="./Addgene_ CRISPR_Cas9 Guide_files/twitter-icon.png" alt="Addgene Twitter Feed" width="30" height="32">
        </a>
        <a href="https://www.youtube.com/user/addgenemedia" target="_blank" rel="noopener noreferrer">
          <img src="./Addgene_ CRISPR_Cas9 Guide_files/youtube-icon.png" alt="Addgene YouTube Feed" width="34" height="30">
        </a>
         <a href="https://www.linkedin.com/company/addgene" target="_blank" rel="noopener noreferrer">
          <img src="./Addgene_ CRISPR_Cas9 Guide_files/linkedin-icon.png" alt="Addgene Linked-In Profile" width="30" height="32">
        </a>
      </div>

      <div id="bluegene-in-footer">
        <img src="./Addgene_ CRISPR_Cas9 Guide_files/bluegene-no-legs.png" alt="Addgene bluegene mascot">
      </div>
    </div>
  </div>

  <div id="addgene-ocean">
    <div class="container">
      <div class="row">
      <div class="footer-left col-xs-3">
        <p>
          Addgene is a nonprofit plasmid repository. We archive and
          distribute high quality plasmids from your colleagues.
        </p>
      </div>
      <div class="footer-right col-xs-9">
        <ul class="footer-nav list-inline">
          
          <li><a href="https://www.addgene.org/contact/">Contact</a></li>
          
          <li><a href="https://www.addgene.org/faq/">FAQ</a></li>
          
          <li><a href="https://www.addgene.org/terms-of-use/">Terms of Use</a></li>
          
          <li><a href="https://www.addgene.org/privacy/">Cookies &amp; Privacy Policy</a></li>
          
          <li><a href="https://www.addgene.org/sitemap/">Site Map</a></li>
          
          <li><a href="http://blog.addgene.org/">Blog</a></li>
        </ul>

        <p class="address">
          Addgene
          • 75 Sidney St, Suite 550A
          • Cambridge, MA 02139
          • +1 (617) 225-9000
          • <a href="mailto:help@addgene.org">help@addgene.org</a>
        </p>
      </div>
    </div>
  </div>
  </div>
</div>
</footer>

        

        
        <!-- site-wide JS, compressed -->
        <script type="text/javascript" src="./Addgene_ CRISPR_Cas9 Guide_files/0b0b7b693fba.js.download"></script>


        
        

        <!-- external JS -->
        
        
        
        
        

        
        <!--- external JS that must be run as async defer -->
        
        
        


        <!-- static JS that can't be combined with other scripts -->
        
        
        


        <!-- static JS for this page -->
        
        
        
        
        
        
        
        <script type="text/javascript" src="./Addgene_ CRISPR_Cas9 Guide_files/cc0ff673d012.js.download"></script>
        
        


        
        <!-- media.js -->
        
        

        



        <!-- pagejs block -->
        
        
        
        







<!-- Third-party tracking Javascript. -->


<!-- Google Analytics Block -->







<!-- Third-party Javascript. This inserts the Zendesk "Help" widget.  -->











        
        
        <!-- Sekizai "js" block start -->
        
<script id="html-chunk-plugin--js" type="text/javascript">
  // This keeps the nav panel from heading north of the <hr> and keeps the nav panel from heading past the footer

$('#crispr-nav-panel').affix({
    offset: {
        top: $('#crispr-nav-panel').offset().top,
        bottom: ($('footer').outerHeight(true)) + 100
    }
});

$('body').scrollspy({ 
    target: '#scrollspy-div' 
    
});

/* $(window).resize(function() {
  if ( $(window).height() < 800) {
    $('#sidebar').addClass('beAbsolute').removeClass('beFixed');
  } else {
    $('#sidebar').addClass('beFixed').removeClass('beAbsolute');
  }
}); */
</script>


        <!-- Sekizai "js" block end -->
    <script>/* <![CDATA[ */(function(d,s,a,i,j,r,l,m,t){try{l=d.getElementsByTagName('a');t=d.createElement('textarea');for(i=0;l.length-i;i++){try{a=l[i].href;s=a.indexOf('/cdn-cgi/l/email-protection');m=a.length;if(a&&s>-1&&m>28){j=28+s;s='';if(j<m){r='0x'+a.substr(j,2)|0;for(j+=2;j<m&&a.charAt(j)!='X';j+=2)s+='%'+('0'+('0x'+a.substr(j,2)^r).toString(16)).slice(-2);j++;s=decodeURIComponent(s)+a.substr(j,m-j)}t.innerHTML=s.replace(/</g,'&lt;').replace(/\>/g,'&gt;');l[i].href='mailto:'+t.value}}catch(e){}}}catch(e){}})(document);/* ]]> */</script>

<div><iframe data-reactroot="" id="helpCenterForm" class="zEWidget-helpCenterForm" style="border: none; background: transparent; z-index: 999999; transform: translateZ(0px); position: fixed; opacity: 0; right: 0px; bottom: 0px; width: 357px; margin-left: 15px; margin-right: 15px; margin-top: 0px; height: 261px; transition-property: none; transition-timing-function: unset; top: -9999px;" src="./Addgene_ CRISPR_Cas9 Guide_files/saved_resource(1).html"></iframe></div><div><iframe data-reactroot="" id="ticketSubmissionForm" class="zEWidget-ticketSubmissionForm" style="border: none; background: transparent; z-index: 999999; transform: translateZ(0px); position: fixed; opacity: 0; right: 0px; bottom: 0px; width: 357px; margin-left: 15px; margin-right: 15px; margin-top: 0px; height: 476px; transition-property: none; transition-timing-function: unset; top: -9999px;" src="./Addgene_ CRISPR_Cas9 Guide_files/saved_resource(2).html"></iframe></div><div><iframe data-reactroot="" id="launcher" class="zEWidget-launcher zEWidget-launcher--active" style="border: none; background: transparent; z-index: 999998; transform: translateZ(0px); position: fixed; opacity: 1; right: 0px; bottom: 0px; width: 111px; height: 47px; margin: 10px 20px;" src="./Addgene_ CRISPR_Cas9 Guide_files/saved_resource(3).html"></iframe></div><div><iframe data-reactroot="" id="nps" class="zEWidget-nps" style="border: none; background: transparent; z-index: 2147483647; transform: translateZ(0px); position: fixed; opacity: 0; right: 0px; bottom: 0px; left: 50%; margin-left: -310px; margin-bottom: 15px; width: 640px; margin-top: 0px; height: 177px; transition-property: none; transition-timing-function: unset; top: -9999px;" src="./Addgene_ CRISPR_Cas9 Guide_files/saved_resource(4).html"></iframe></div><div><iframe data-reactroot="" id="ipm" class="zEWidget-ipm" style="border: none; background: transparent; z-index: 2147483647; transform: translateZ(0px); position: fixed; opacity: 0; right: 0px; top: -230px; margin-right: 15px; margin-top: 0px; height: 180px; width: 395px; transition-property: none; transition-timing-function: unset;" src="./Addgene_ CRISPR_Cas9 Guide_files/saved_resource(5).html"></iframe></div></body></html>